ci/hoover.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Regular maintenance performed on a buildkite agent to control disk usage
+#
+
+echo --- Delete all exited containers first
+(
+  set -x
+  exited=$(docker ps -aq --no-trunc --filter "status=exited")
+  if [[ -n "$exited" ]]; then
+    docker rm "$exited"
+  fi
+)
+
+echo --- Delete untagged images
+(
+  set -x
+  untagged=$(docker images | grep '<none>'| awk '{ print $3 }')
+  if [[ -n "$untagged" ]]; then
+    docker rmi "$untagged"
+  fi
+)
+
+echo --- Delete all dangling images
+(
+  set -x
+  dangling=$(docker images --filter 'dangling=true' -q --no-trunc | sort | uniq)
+  if [[ -n "$dangling" ]]; then
+    docker rmi "$dangling"
```

Comment:
looks to me like dangling is multiple strings...  when you pass "$dangling" to docker rmi, you pass a single argument

consider
dangling=( $(docker ...) )
docker rmi "${dangliing[@]}"